error.go

```diff
@@ -0,0 +1,23 @@
+package tenantcluster
```

Comment:
Code is unchanged from `guestcluster`. Only changes are renaming guest to tenant.